tests/phpunit/tests/meta/bulkAddMetadata.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @group meta
+ *
+ * @covers bulk_add_metadata
+ */
+class Tests_Meta_BulkAddMetadata extends WP_UnitTestCase {
```

Comment:
Minor: each of these tests is using postmeta. I think using other meta as well would be good to make sure that it's not just postmeta but the other meta as well that is also tested